ci/cron/daily-compat.yml

```diff
@@ -196,3 +196,61 @@ jobs:
         env:
           GCRED: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
       - template: ../daily_tell_slack.yml
+
+  - job: blackduck_scan
+    timeoutInMinutes: 1200
+    pool:
+      name: linux-pool
+      demands: assignment -equals default
+    steps:
+      - checkout: self
+      - bash: ci/dev-env-install.sh
+        displayName: 'Build/Install the Developer Environment'
+      - bash: ci/configure-bazel.sh
+        displayName: 'Configure Bazel'
+        env:
+          IS_FORK: $(System.PullRequest.IsFork)
+          # to upload to the bazel cache
+          GOOGLE_APPLICATION_CREDENTIALS_CONTENT: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)        
+      - template: ../bash-lib.yml
+        parameters:
+          var_name: bash_lib
+      - bash: ./build.sh "_$(uname)"
+        displayName: 'Build'
+      - bash: |
+          set -euo pipefail
+          eval "$(./dev-env/bin/dade-assist)"
```

Comment:
Should we checkout master here? Otherwise, given the very time-dependent nature of this test, this will make rerunning old commits pretty much impossible.

Reply:
I’m confused, daily compat only runs on master?

Reply:
I mean `git fetch && git checkout origin/master`, because scanning for outdated deps in older commits makes no sense.

Reply:
Ah I see, seems sensible to me.

Reply:
@garyverhaegen-da not sure we want to force this though -- shouldn't we leave it to run on the branch in play, also to be able to run on branches prior to master if we need to make changes or tweaks to the script?

Reply:
This is not running on any branch at all, this is a daily run on master.